requirements/prod.in

```diff
@@ -0,0 +1,28 @@
+click
+coloredlogs
+cryptography
+Faker
+fs
+github3.py
+jinja2
+keyring
+lxml
+pydantic
+PyJWT
+pytz
+requests
+requests-futures
+robotframework==3.2.2
```

Comment:
```suggestion
robotframework<4
```

That way we still get minor & patch updates when we recompile, but not the major update until we've reviewed it.